### ISSUE TITLE
Parameter issues

### ISSUE THREAD

**Issue by PatrickStar228** [YC-Liang/DRT#10]
Hello Author,

I am encountering some issues with the results on the three datasets—Test100, Rain100L, and Rain100H—not matching the descriptions in the paper. I adjusted parameters such as batch_size, training_image_size, epochs, and learning rate decay strategies, which allowed Rain100L to achieve the expected PSNR and SSIM. However, I have not yet found suitable parameters for Test100 and Rain100H. Could you please provide the corresponding training parameters?
